docs/index.html

```diff
@@ -109,17 +109,17 @@
-<div class="fragment"><div class="line">cart.LineItems.Get(firstProduct, selectedOptions);</div><div class="line">cart.LineItems.Delete(firstProduct, selectedOptions);</div></div><!-- fragment --><h2>Native web view checkout</h2>
-<p>After creating an instance of <code>Cart</code> and adding items to it, you can use the <code>CheckoutWithNativeWebView</code> method to start a native modal overlay on top of your game with a web view that contains the checkout for the cart.</p>
-<p><code>CheckoutWithNativeWebView</code> takes in 3 callback parameters:</p>
+<div class="fragment"><div class="line">cart.LineItems.Get(firstProduct, selectedOptions);</div><div class="line">cart.LineItems.Delete(firstProduct, selectedOptions);</div></div><!-- fragment --><h2>Web view checkout</h2>
+<p>After creating an instance of <code>Cart</code> and adding items to it, you can use the <code>CheckoutWithWebView</code> method to start a native modal overlay on top of your game with a web view that contains the checkout for the cart.</p>
```

Comment:
Just this last description needs to change: `method to start a native modal overlay on top of your game with a web view that contains the checkout for the cart`

It's no longer a native modal overlay or a web view